contracts/ligo/src/swaps/README.md

```diff
@@ -0,0 +1,8 @@
+# Swaps contract
+
+This contract allows a participant to offer FA2 tokens exchange that another participant can accept.
+
+Specification: https://hackmd.io/iArxW3FyR_e85JNPYvuQhQ.
+
+In case you are implementing a client for this contract, please see
+"Notes to middleware implementors" section in the spec mentioned above.
```

Comment:
Even tho the contract is attached I'd put a `Makefile` in this dir as well to generate it.
I think it would be quite useful and it should be mentioned in this README as well.

I suggest to also add a target for the storage using `init_storage` and `ligo`'s `compile_storage`.
Note: this being very simple perhaps we should just output the result to stdout, what do you think?

Reply:
Oh, that's a really good idea, taking into account that I now own a separate folder, it's worth adding a Makefile.

Yep, and storage is also worth printing, stdout really seems just fine.

Reply:
Hmmm, it looks like most of the `ligo` commands except for `compile-contract` do not work when `#include` is used, great. Is it fine if I do not export the initial storage here?

Reply:
There is a caveat because you run LIGO in a docker container and mount the volumes.
Your work directory (the one used to invoke LIGO from) has to be a parent to all directories that contain your source code.
I. e. if the contract includes some file from one of the parents directory: `#include "../.<somefile>"`, you cannot invoke LIGO from the same directory where that contract code is.

Reply:
Oh, looks like mixed up some things, I meant to say `#import`, not `#include`.

Currently, I'm using just a static Linux binary. Everything really works well if my contract uses only `#include` pragma, but for `#import` there seem to be several bugs yet, probably this is just one of them.